### PR TITLE
Update laurian.js

### PR DESCRIPTION
**Pull request by maboa**
now with added meh

### CODE REVIEW

**Review by Laurian — CHANGES_REQUESTED** [review]
meh

File: scripts/laurian.js [hunk 1]
```diff
@@ -40,4 +43,4 @@ $(function() {
 		$("#question").show();
 		$("#answer").hide();
 	});
-});
\ No newline at end of file
+});
```
Comment (slifty) [inline scripts/laurian.js]: Please break this into a separate whitespace commit

**Review by slifty — CHANGES_REQUESTED** [review]

**Comment by Laurian** [comment]
meh

File: scripts/laurian.js (hunk 1 above)
Comment (maboa) [reply scripts/laurian.js:46]: WHAT!?

Comment (maboa) [reply scripts/laurian.js:46]: IT WAS LIKE THAT WHEN I FOUND IT.

**Pull request merged**